website/static/js/fangorn.js

```diff
@@ -1935,18 +1935,33 @@ var FGItemButtons = {
         if (tb.options.placement !== 'fileview') {
             if (window.File && window.FileReader && item.kind === 'folder' && item.data.provider && item.data.permissions && item.data.permissions.edit) {
                 rowButtons.push(
-                    m.component(FGButton, {
-                        onclick: function(event) {_uploadEvent.call(tb, event, item); },
-                        icon: 'fa fa-upload',
-                        className : 'text-success'
-                    }, 'Upload'),
                     m.component(FGButton, {
                         onclick: function () {
                             mode(toolbarModes.ADDFOLDER);
                         },
                         icon: 'fa fa-plus',
                         className: 'text-success'
-                    }, 'Create Folder'));
+                    }, 'Create Folder')
+                );
+                if(window.contextVars.node.storageLimitsStatus) {
```

Comment:
Drive-by: Did you disable the dropzone for OSF Storage when the limit was exceeded?

Reply:
It doesn't have a visual indicator that the dropzone is disabled, but if you try to drop it produces a growlbox with message, (or it will once WB returns a 409 as the response).

Reply:
If possible, it would be better to disable the dropzone, to keep people from attempting to upload files, which will save bandwidth costs, and keep the users happier by not wasting their time. Don't go through heroics for this, but it would be better.

Reply:
Yeah I tried a few different things, but it's too difficult to do this dynamically so that uploading will work once something is deleted and storage usage goes down.